sidebars.js

```diff
@@ -106,9 +106,16 @@ module.exports = {
                       label: "Available Targeting",
                       href: "https://firefox-source-docs.mozilla.org/browser/components/newtab/content-src/asrouter/docs/targeting-attributes.html",
                     },
-                    "messaging/mobile-messaging"
-                  ]
+                  ],
                 },
+                {
+                  "type": "category",
+                  "label": "Messaging Experiments on Firefox for Mobile",
+                  items: [
+                    "messaging/mobile-messaging",
+                    "deep-dives/mobile/mobile-behavioral-targeting",
+                  ]
```

Comment:
I agree this should probably be here, I will note however that now when you navigate to this page the sidebar shows that you are accessing two of the page and opens both their trees.